### PR TITLE
Added flags '--only-pg-support' and '-v' or '--version' to the installer script

### PR DESCRIPTION
**Pull request by ShivanshGahlot**

### CODE REVIEW

File: installer_scripts/install-yb-voyager [hunk 1]
```diff
@@ -6,19 +6,37 @@ export LC_ALL=en_US.UTF-8
...
-		# Install from locally checked out branch.
-		VERSION=local
-	else
-		echo "ERROR: Invalid argument: '$1'"
-		exit 1
-	fi
+OPTS=$(getopt -o v: --long version:,only-pg-support -n 'install-yb-voyager' -- "$@")
+
+if [ $? != 0 ]; then 
+	echo "Unable to process the options. Exiting..." >&2;
```
Comment (amit-yb) [inline installer_scripts/install-yb-voyager:12]: Does the user get the exact error why the script couldn't process options?

File: installer_scripts/install-yb-voyager [hunk 2]
```diff
@@ -6,19 +6,37 @@ export LC_ALL=en_US.UTF-8
...
+		-v | --version ) 
+			if [ "$2" == local ]; 
+			then 
+				# Install from locally checked out branch.
+				VERSION="$2"; shift 2;
+			else
+				echo "ERROR: Invalid version"
+				exit 1
+			fi 
+			;;
```
Comment (amit-yb) [inline installer_scripts/install-yb-voyager:32]: Let's have a single boolean flag named `--install-from-local-source`. If passed on CLI, set the `VERSION` to `local`.

File: installer_scripts/install-yb-voyager [hunk 3]
```diff
@@ -6,19 +6,37 @@ export LC_ALL=en_US.UTF-8
...
+		--only-pg-support ) ONLY_PG="true"; shift ;;
```
Comment (amit-yb) [inline installer_scripts/install-yb-voyager:33]: Move the option processing into separate lines:
```
  --only-pg-support )
    ONLY_PG="true"
    shift
    ;;
```

File: installer_scripts/install-yb-voyager [hunk 4]
```diff
@@ -6,19 +6,37 @@ export LC_ALL=en_US.UTF-8
...
+		-- ) shift; break ;;
+		* ) break ;;
```
Comment (amit-yb) [inline installer_scripts/install-yb-voyager:35]: Just for my understanding, why is this required?

File: installer_scripts/install-yb-voyager [hunk 5]
```diff
@@ -108,19 +126,23 @@ centos_main() {
...
+		centos_install_oracle_instant_clients
+		update_yb_voyager_bashrc
+		install_ora2pg
+	fi
 	install_yb_voyager
 	create_yb_session_vars_file
-	create_base_ora2pg_conf_file
+	if [ $ONLY_PG == "false" ] ; then
+		create_base_ora2pg_conf_file
+	fi
```
Comment (amit-yb) [inline installer_scripts/install-yb-voyager:145]: Merge this if block into the earlier if block by moving line 144 after line 139.

File: installer_scripts/install-yb-voyager [hunk 6]
```diff
@@ -132,20 +154,24 @@ ubuntu_main() {
...
+		ubuntu_install_oracle_instant_clients
+		update_yb_voyager_bashrc
+		install_ora2pg
+	fi
 	install_yb_voyager
 	create_yb_session_vars_file
-	create_base_ora2pg_conf_file
+	if [ $ONLY_PG == "false" ] ; then
+		create_base_ora2pg_conf_file
+	fi
```
Comment (amit-yb) [inline installer_scripts/install-yb-voyager:174]: Same as above.

File: installer_scripts/install-yb-voyager (hunk 2 above)
Comment (ShivanshGahlot) [reply installer_scripts/install-yb-voyager:32]: Done

File: installer_scripts/install-yb-voyager (hunk 3 above)
Comment (ShivanshGahlot) [reply installer_scripts/install-yb-voyager:33]: Done

File: installer_scripts/install-yb-voyager (hunk 5 above)
Comment (ShivanshGahlot) [reply installer_scripts/install-yb-voyager:145]: Actually I did this only earlier. But the `/etc/yb-voyager` is created in the `create_yb_session_vars_file`. I have now added a` mkdir -p /etc/yb-voyager` statement in `create_base_ora2pg_conf_file` too. Since in case of macos we only run `create_yb_session_vars_file`, I can't remove the `mkdir -p /etc/yb-voyager` from that function.

File: installer_scripts/install-yb-voyager (hunk 1 above)
Comment (ShivanshGahlot) [reply installer_scripts/install-yb-voyager:12]: The user gets an error if a wrong option is passed. I think this statement is unnecessary. I found it in a template on stackoverflow. I have removed it for now.
Also a small finding. getopt doesn't match the whole string. For example, the result would be the same if either `--only-pg-support` or `--only `is passed.

File: installer_scripts/install-yb-voyager (hunk 4 above)
Comment (ShivanshGahlot) [reply installer_scripts/install-yb-voyager:35]: This is also unnecessary. I have removed it for now. I tried running without this and it worked fine.

**Review by amit-yb — APPROVED** [review]
Good work! 👍
